src/templates/static/about/binary_in_numbers.jsx

```diff
@@ -21,31 +21,31 @@ const BinaryInNumbers = () => (
             <Numbers
                 className='client'
                 header={it.L('Active trading clients')}
-                text={it.L('[_1]’s number of active clients has risen nearly 5-fold in the last 5 years.', it.website_name)}
+                text={it.L('Our active client base is expected to reach over 100,000 total clients this year.')}
                 image='chart-active-trading-clients'
             />
             <Numbers
                 className='transaction'
                 header={it.L('Number of transactions')}
-                text={it.L('In last two years alone, over 276 million contracts were bought and sold on our platform.')}
+                text={it.L('We’re projected to register over 197 million transactions this year, bringing the total number of contracts that were bought and sold on our platform since inception to over 789 million.')}
                 image='chart-num-transactions'
             />
             <Numbers
                 className='turnover'
                 header={it.L('Turnover')}
-                text={it.L('We\'ve generated over USD 3 billion in turnover since inception, thanks to a rapid growth in business from 2010 onwards.')}
+                text={it.L('We’re expected to generate over USD 1 billion in turnover this year, bringing our total turnover since inception to over USD 5.7 billion.')}
                 image='chart-turnover'
             />
             <Numbers
                 className='withdrawal'
                 header={it.L('Client withdrawals')}
-                text={it.L('Client withdrawals have grown nearly 8-fold over the last 5 years, to nearly USD 120 million.')}
+                text={it.L('Client withdrawals prior to this year amounted to almost USD 500 million. More than USD 140 million worth of client withdrawals is expected this year.')}
                 image='chart-client-withdrawals'
             />
             <Numbers
                 className='employee'
                 header={it.L('Number of employees and contractors')}
-                text={it.L('We\'ve grown in size over the years -- both in terms of manpower and offices. We currently have over 130 employees across four offices in three countries: Malaysia, Malta and Japan.')}
+                text={it.L('We’ve grown in size over the years –– both in terms of manpower and offices. We currently have over 130 employees across four offices in Malaysia, Malta, and Japan. Further growth in headcount is expected this year to cater to the increased demand in our products and services.')}
```

Comment:
Please change "We currently have over 130 employees" to "We currently have over 150 employees"